WalletWasabi.Gui/Controls/WalletExplorer/ReceiveTabView.xaml

```diff
@@ -30,7 +30,7 @@
           </Grid>
         </DockPanel>
         <Grid ColumnDefinitions="300, *, 50" Margin="5 10 5 0" Grid.Row="1">
-          <TextBlock Text="Label:" />
+          <TextBlock Text="Involved Parties:" />
```

Comment:
Could you screenshot this?

Reply:
where exactly is it? I can't tell by looking at the code, is it not the textbox in the receive tab that is screenshot above?

Reply:
I don't know.

Reply:
I think that is the title of the columns but I am not sure either which is not displayed or something.

Reply:
Yes, I think so.

Reply:
Why do we give those columns a text if we do not display it? @jmacato